Update section-01.md

joshc0044 opened the pull request:
added some good video game related video links #1

joshc0044 committed:
Update section-01.md

added some good video game related video links

omarsar commented:
@joshc0044 thanks for the PR. 

I notice these links are more specific to game-related content. Very interesting by the way! :)

However, for this particular list, I was aiming to share links that contain more general introductions to deep learning or neural networks.

joshc0044 commented:
@omarsar Gotcha, hopefully some of these resources can be useful as case studies in upcoming sections, I'll look for some introductory video game related material and amend this pr

joshc0044 committed:
Update section-01.md

removed the specific case studies, added video with more general overview of various uses of machine learning by ubisoft
